unit-tests/live/options/test-rgb-options-metadata-consistency.py

```diff
@@ -7,6 +7,11 @@
 import pyrealsense2 as rs
 from rspy import test
 
+def close_resources(sensor):
+    if not sensor.get_active_streams():
```

Comment:
You mean the other way around, maybe? if no streams then no need to close, no?

Reply:
Yes, not my day.
Will fix it tomorrow.